_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1283,28 +3275,27 @@ To limit the number of flow logs, you may want to use the
 link:https://github.com/gruntwork-io/cloud-nuke[`cloud-nuke defaults-aws`] command. It will remove the default VPC from
 all regions in an account, saving you the hassle of creating flow logs in each default VPC.
 
-[[configure_monitoring]]
```

Comment:
I feel like in this section we should also include the IAM Access Analyzer & the Security Hub too. My thinking is that they're also monitoring tools for any security issues. Curious to get more thoughts on this :)